src/DataFactory/DataFactoryV2/Changelog.md

```diff
@@ -20,6 +20,10 @@
 
 ## Upcoming Release
 
+## Version 1.0.3
```

Comment:
@sanjkuma please remove this header and place the below text under the `Upcoming Release` header

Reply:
Done.